Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public class LoginEntryDialogueView: UIView {
```

Comment:
Does this class need to be public?

Reply:
I guess it doesn't really have to be. I can fix that